### PR TITLE
Update html_baseurl to iot

### PR DESCRIPTION
**Pull request by tzach**

### CODE REVIEW

**Comment by tzach** [comment]
@dgarcia360 please review

File: docs/conf.py [hunk 1]
```diff
@@ -91,7 +91,7 @@
 htmlhelp_basename = 'ScyllaDocumentationdoc'
 
 # URL which points to the root of the HTML documentation. 
-html_baseurl = 'https://care-pet.docs.scylladb.com'
+html_baseurl = 'https://iot.docs.scylladb.com'
```
Comment (dgarcia360) [inline docs/conf.py:94]:
```suggestion
html_baseurl = 'https://iot.scylladb.com'
```

**Review by dgarcia360 — COMMENTED** [review]
The subdomain is  https://iot.scylladb.com/, not https://iot.docs.scylladb.com/

**Comment by dgarcia360** [comment]
LGTM!

**Pull request merged**